<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkDicomEditorControls</class>
 <widget class="QWidget" name="QmitkDicomEditorControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>696</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="LocalStorageButton">
           <property name="text">
            <string>Local Storage</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="CDButton">
           <property name="text">
            <string>Import CD</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="FolderButton">
           <property name="text">
            <string>Import Folder</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="QueryRetrieveButton">
           <property name="text">
            <string>Query Retrieve</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QFrame" name="internalDataFrame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
             <item>
              <widget class="QGroupBox" name="groupBox">
               <property name="title">
                <string>Sorting Tags</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QListView" name="listView"/>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <property name="leftMargin">
                     <number>9</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>9</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QPushButton" name="addSortingTagButton">
                      <property name="maximumSize">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>new</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="deleteSortingTagButton">
                      <property name="maximumSize">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>delete</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <spacer name="horizontalSpacer">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="localFilesGroupBox">
               <property name="title">
                <string>Local Dicom Storage</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_11">
                <item>
                 <widget class="QFrame" name="frame_2">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <item>
                    <widget class="QPushButton" name="viewInternalDataButton">
                     <property name="enabled">
                      <bool>true</bool>
                     </property>
                     <property name="text">
                      <string>View</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pushButton">
                     <property name="enabled">
                      <bool>true</bool>
                     </property>
                     <property name="text">
                      <string>Delete</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer_4">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="QueryLayout" stretch="1,0">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>12</number>
                  </property>
                  <property name="rightMargin">
                   <number>12</number>
                  </property>
                  <item>
                   <widget class="QTreeView" name="InternalDataTreeView">
                    <property name="alternatingRowColors">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="searchOptionLayout">
                    <property name="spacing">
                     <number>6</number>
                    </property>
                    <item>
                     <widget class="QDockWidget" name="InternalSearchDockWidget">
                      <property name="floating">
                       <bool>false</bool>
                      </property>
                      <property name="features">
                       <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
                      </property>
                      <property name="windowTitle">
                       <string/>
                      </property>
                      <widget class="QWidget" name="dockWidgetContents">
                       <layout class="QVBoxLayout" name="verticalLayout_5">
                        <item>
                         <widget class="ctkDICOMQueryWidget" name="SearchOption" native="true">
                          <property name="minimumSize">
                           <size>
                            <width>0</width>
                            <height>0</height>
                           </size>
                          </property>
                         </widget>
                        </item>
                       </layout>
                      </widget>
                     </widget>
                    </item>
                    <item>
                     <spacer name="VerticalSpacer">
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>20</width>
                        <height>40</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <property name="leftMargin">
                   <number>18</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>9</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QPushButton" name="addSortingTagButton_2">
                    <property name="enabled">
                     <bool>true</bool>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>add</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="deleteSortingTagButton_2">
                    <property name="enabled">
                     <bool>true</bool>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>delete</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
         <zorder>groupBox</zorder>
         <zorder>localFilesGroupBox</zorder>
         <zorder>internalDataFrame</zorder>
        </widget>
        <widget class="QWidget" name="importWidgetPage" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QGroupBox" name="externalFilesGroupBox">
            <property name="title">
             <string>External Dicom Data</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <widget class="QFrame" name="itemOperationsFrame">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QPushButton" name="viewExternalDataButton">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="text">
                   <string>View</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="downloadButton">
                  <property name="text">
                   <string>Download</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="cancelButton">
                  <property name="text">
                   <string>Cancel</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="QueryLayout_2" stretch="1,0">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>12</number>
               </property>
               <property name="rightMargin">
                <number>12</number>
               </property>
               <item>
                <widget class="QTreeView" name="ExternalDataTreeView">
                 <property name="alternatingRowColors">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QVBoxLayout" name="searchOptionLayout_2">
                 <property name="spacing">
                  <number>6</number>
                 </property>
                 <item>
                  <widget class="QDockWidget" name="ExternalSearchDockWidget">
                   <property name="floating">
                    <bool>false</bool>
                   </property>
                   <property name="features">
                    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
                   </property>
                   <property name="windowTitle">
                    <string/>
                   </property>
                   <widget class="QWidget" name="dockWidgetContents_2">
                    <layout class="QVBoxLayout" name="verticalLayout_6">
                     <item>
                      <widget class="ctkDICOMQueryWidget" name="SearchOption_2" native="true">
                       <property name="minimumSize">
                        <size>
                         <width>0</width>
                         <height>0</height>
                        </size>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </widget>
                 </item>
                 <item>
                  <spacer name="VerticalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="queryRetrieveWidgetPage" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="ctkDICOMQueryRetrieveWidget" name="m_ctkDICOMQueryRetrieveWidget" native="true"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ctkDICOMQueryRetrieveWidget</class>
   <extends>QWidget</extends>
   <header>ctkdicomqueryretrievewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkDICOMQueryWidget</class>
   <extends>QWidget</extends>
   <header>ctkdicomquerywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
